login/user_claims.go

```diff
@@ -87,5 +89,19 @@ func match(userInfo model.UserInfo, entry userFileEntry) bool {
 	if entry.Origin != "" && entry.Origin != userInfo.Origin {
 		return false
 	}
+	if len(entry.Groups) > 0 {
```

Comment:
could you please add a test for this block?

Reply:
It is now resolved at commit https://github.com/tarent/loginsrv/pull/92/commits/18878164d7cc7e67a7f69c8cb662c604e9d8e5bc